app/utils/sanity/start.go

```diff
@@ -78,10 +82,23 @@ var (
 		},
 	}
 
-	checkRangeItems = []rangeItem{{
-		enumRange: []int{int(state.DeliverTxsExecModeSerial), state.DeliverTxsExecModeParallel},
-		name:      state.FlagDeliverTxsExecMode,
-	},
+	checkRangeItems = []rangeItem{
+		{
+			enumRange: []int{int(state.DeliverTxsExecModeSerial), state.DeliverTxsExecModeParallel},
+			name:      state.FlagDeliverTxsExecMode,
+		},
+	}
+
+	conflictPairOpts = []conflictPairWithOption{
+		{
+			conflictPair: conflictPair{
+				configA: boolItem{name: iavl.FlagIavlEnableFastStorage, value: true},
```

Comment:
value can rename expect or expect name

Reply:
both boolItem/stringItem's value have renamed to expect